raitools/raitools/raianalyzer/constants.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+from enum import Enum
+
+
+class ModelTask(str, Enum):
+    """Provide model task constants. Can be 'classification' or 'regression'.
+    """
+
+    Classification = 'classification'
```

Comment:
BinaryClassification, surely?

Reply:
we automatically recognize this in our explainers, I'm not sure about other libraries like fairlearn, but if they need this specified I can add it

Reply:
We only have metrics for binary classification and regression (and the probability ones). The meaning of things like 'false positive rate' start becoming a little fuzzy when you have multiclass classification. Again, perhaps a PM question.

Reply:
Asked PM to clarify, waiting for PM reply.